content/building/build-versioning.md

```diff
@@ -20,12 +20,12 @@ Here are some examples how you can increment the app version using Codemagic's r
 --build-name=1.0.0 --build-number=$BUILD_NUMBER
 ```
 
-## Fetching build number from pubsec.yaml
+## Fetching build name and build number from pubsec.yaml
 
 Add the following build argument:
 
 ```bash
---build-number=$(yq e .version pubspec.yaml)
+--build-name=$(yq e .version pubspec.yaml | cut -d "+" -f1) --build-number=$(yq e .version pubspec.yaml | cut -d "+" -f2) 
```

Comment:
In case version does not have the `+int` modifier, then both build name and build version will be the same `x.y.z`, which will result in invalid build number, doesn't it?

Reply:
yes so maybe it is good to add the example as well. The[ official flutter docs](https://flutter.dev/docs/deployment/android#updating-the-apps-version-number) describe that `x.y.z` is used for version and `+w` is used for buildnumber.